openwisp_users/migrations/0003_default_organization.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.1.1 on 2018-09-16 12:20
+
+from django.db import migrations
+from django.conf import settings
+
+
+def set_default_organization_uuid(apps, schema_editor):
+    """
+    Get default organzation or create a default
+    organization and set settings.DEFAULT_ORG_UUID
+    """
+    organization = apps.get_model('openwisp_users', 'organization')
+    try:
+        default_organization = organization.objects.get(
+            slug=settings.DEFAULT_ORG_SLUG)
+
+    except (AttributeError, organization.DoesNotExist):
+        default_organization = organization.objects.create(
+            name='default',
+            slug='default',
+            is_active=False,
+            description='This is the default organization. '
+            'It was created because no organization was set as '
+            'default during setup. You can simply rename it '
+            'to your preferred organization name.'
+        )
+
+    settings.DEFAULT_ORG_SLUG = default_organization.slug
+    settings.DEFAULT_ORG_UUID = default_organization.pk
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('openwisp_users', '0002_auto_20180508_2017'),
+    ]
+
+    operations = [
+        migrations.RunPython(set_default_organization_uuid),
```

Comment:
you need to add `, reverse_code=migrations.RunPython.noop` or it won't be possible to run the migrations backward